src/utils/motivation.js

```diff
@@ -1,38 +1,47 @@
+const messages = {
+  0: [
+    'Not lookin\' so good yet, but you have time, right?',
+    'Go ahead, make your first PR!',
+    'You can do better!',
+    'They say it\'s never too late to start!'
+  ],
+  1: [
+    'That wasn\'t so hard after all, was it?',
+    'That\'s right. It\'s 20% progress already.',
+    'Wonderful! Keep bustin\' those issues!',
+    'Hope you enjoyed that one. Go ahead!'
+  ],
+  2: [
+    ':pray: You\'re doing great, keep going!',
+    ':pray: Nice job, 3 more left.',
+    ':pray: 2 done, 3 to go!'
+  ],
+  3: [
+    ':sunglasses: Just two more!',
+    ':sunglasses: You ain\'t gonna stop now, huh?',
+    ':sunglasses: Two more, come on!',
+    ':sunglasses: 3 done, lovely!',
+  ],
+  4: [
+    ':raised_hands: Just one more!',
+    ':raised_hands: Squeeze that last one out!',
+    ':raised_hands: Finish line, don\'t stop now!',
+    ':raised_hands: Keep going!'
+  ],
+  5: [
+    ':checkered_flag: Wow! You made it! Nice job, mate!',
+    ':checkered_flag: Congrats! You have beaten Hacktoberfest!',
+    ':checkered_flag: Done! Now sit back and wait for those stickers to arrive.',
+    ':checkered_flag: Coolio! You did all 5 PRs. Grab a beer and celebrate!'
+  ]
+};
+
+export const numberOfNeededPullRequests = Object.keys(messages).length - 1;
```

Comment:
@peppertom I'm not entirely convinced `utils/motivation.js` should have the responsibility to hold such a variable as the `numberOfNeededPullRequests`, but let's go with it anyway, LGTM.

Reply:
Agree on that @wolfika , updated that in a new PR, and also fixed the slice fn's second parameter of fixed 4 items.